x-pack/plugins/fleet/server/routes/security.ts

```diff
@@ -110,6 +112,116 @@ export async function getAuthzFromRequest(req: KibanaRequest): Promise<FleetAuth
   });
 }
 
+// needed from Read routes
+export const readEndpointPackagePrivileges: DeepPartialTruthy<FleetAuthz> = Object.freeze({
```

Comment:
Suggestion: name the constant in upper case: `READ_ENDPINT_PACKAGE_PRIVILEGES` (we normally name constants in upper case using snake_case).
Same for the other const below.